src/SumoCoders/FrameworkCoreBundle/Resources/views/base.html.twig

```diff
@@ -44,16 +44,7 @@
       <script type="text/javascript" src="{{ asset('/bundles/mopabootstrap/js/mopabootstrap-collection.js') }}"></script>
       <script type="text/javascript" src="{{ asset('/assets/js/sumo_plugins.js') }}"></script>
 
-      {% if app.environment == 'dev' %}
```

Comment:
bundle.js is generated with webpack and is minified. If you want to add a new file to bundle, you have to import it in index.js. Examples here: https://github.com/sumocoders/Framework/blob/convert-coffeescript-to-es6/src/SumoCoders/FrameworkCoreBundle/Resources/assets/js/index.js